sources/tanya_main.c

```diff
@@ -3,17 +3,50 @@
 /*                                                        :::      ::::::::   */
 /*   tanya_main.c                                       :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
-/*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
+/*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/09/24 00:41:54 by inikulin          #+#    #+#             */
-/*   Updated: 2024/09/24 00:42:44 by inikulin         ###   ########.fr       */
+/*   Updated: 2024/10/15 17:06:09 by taretiuk         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
-int	main(void)
+static int	interactive(t_param *param)
```

Comment:
Yeah, here in your private main you can do whatever you need for testing. Just make sure to transfer all changes to the main project body.